### PR TITLE
Limit logEntries in aggregated operation proto.

### PR DESCRIPTION
**Pull request by qiwzhang**

### CODE REVIEW

File: src/operation_aggregator.cc [hunk 1]
```diff
@@ -33,6 +33,11 @@ namespace service_control_client {
 
 namespace {
 
+// Each logEntry is about 0.5 KB. LogEntries can only been appended.
+// Not to append too many logEntries in one operation in order to
+// limit the final report size to be < 1MB.
+const int kMaxLogEntries = 100;
```
Comment (chowchow316) [inline src/operation_aggregator.cc]: LogEntries can only be appended.

File: src/report_aggregator_impl.cc [hunk 2]
```diff
@@ -30,10 +30,10 @@ namespace google {
 namespace service_control_client {
 namespace {
 
-// A report can carry many operations, merge multiple report requests
-// into one report request until its number of operations reach this limit.
-// Each report is about 4KB now. Maximum allowed size from server is 1MB.
-const int kMaxOperationsToSend = 100;
+// Service control server limits each report data size to 1MB.
+// Roughly, each operation may have up to 50KB based on maximum of 100
+// aggregated logEntries (each log entry is about 500 bytes).
```
Comment (chowchow316) [inline src/report_aggregator_impl.cc]: How about metrics values?

File: src/operation_aggregator.cc (hunk 1 above)
Comment (qiwzhang) [inline src/operation_aggregator.cc]: Done

File: src/report_aggregator_impl.cc (hunk 2 above)
Comment (qiwzhang) [inline src/report_aggregator_impl.cc]: The data size for all metrics is around 5KB.  If there are many log entries, most of data is from log entries.

**Pull request merged**